src/Utilities/Helpers.js

```diff
@@ -330,8 +334,8 @@ export function sortCves(cves, index, direction) {
 
     const sortedCves = cves.sort(
         ({ cells: aCells }, { cells: bCells }) => {
-            const aCell = aCells[index].value && aCells[index].value || aCells[index].title;
-            const bCell = bCells[index].value && bCells[index].value || bCells[index].title;
+            const aCell = isSet(aCells[index].value) && aCells[index].value || aCells[index].title;
```

Comment:
```suggestion
            const aCell = aCells[index].value || aCells[index].title;
```
Also for other occurrences